change/@fluentui-react-charting-f1e8df5f-9e91-4198-8c23-cd46a8e14e7b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29815&commitId=96b0346d4a06169cffcfb519dfd8d2864bd05244&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-MultiStackBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-MultiStackBarChart.Basic_Absolute - RTL.default.chromium.png|179| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
The PR fixes the issue being reported in the MultiStackedBarChartBasic RTL test case